src/lib/arrays.js

```diff
@@ -0,0 +1,119 @@
+const getNthElement = (index, array) => {
+  return array[index % array.length];
+};
+
+const arrayToCSVString = array => {
+  return array.join(',');
+};
+
+const csvStringToArray = string => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  return array.push(element);
+};
+
+const addToArray2 = (element, array) => {
+  return array.concat(element);
+};
+
+const removeNthElement = (index, array) => {
+  if (!index) {
+    return array.shift();
+  }
+  return array.splice(index, index - 1);
+};
+
+const numbersToStrings = numbers => {
+  return numbers.map(String);
+};
+
+const uppercaseWordsInArray = strings => {
+  return strings.map(words => words.toUpperCase());
+};
+
+const reverseWordsInArray = strings => {
+  strings.map((element, index) => {
+    element = element.split('');
+    element = element.reverse();
+    element = element.join('');
+    strings[index] = element;
+  });
+  return strings;
+  /*
+  const newArray = []
+  strings.forEach(element => {
+    element = element.split('');
+    element = element.reverse();
+    element = element.join('');
+    newArray.push(element);
+  });
+  return newArray;
+  */
```

Comment:
Looks like you didnt finish this one! You were on the right track, just needs a bit of refining:
```
  return strings.map(string => {
    return string
      .split('')
      .reverse()
      .join('');
  });
 ```
I see this didnt flag up as a failing test, presumably you didnt write one as this wouldnt have passed? Not a problem if this is the case, its worth mentioning if there were any issues or stuff that wasnt finished when submitting code for review, or even better adding it to the README